app/src/utils/utils.ts

```diff
@@ -118,3 +125,18 @@ export function displayResults(
   const end = Math.min(page * perPage, numFound);
   return `${start}-${end} of ${numFound}`;
 }
+
+// TODO: right now, whenever a user hits earch, the url is created with all params including the defaults.
+// Perhaps the query string should only include the values that are not defaults.
+export const createQueryStringFromObject = (paramObj) => {
+  const params = new URLSearchParams();
+  Object.keys(paramObj).map((name, value) => {
+    params.set(
+      name.toString(),
+      name === "page" ? paramObj[name].toString() : paramObj[name]
+    );
+  });
+  return params.toString();
+};
+
+// paramObj options:
```

Comment:
Instead of listing it down in a comment, add types to define the properties in the `paramObj`.